Utilites/Shell++/v1/task.json

```
@@ -0,0 +1,103 @@
+{
+  "id": "10f1f9a1-74b0-47ab-87bf-e3c9c68e8b0d",
+  "name": "Shellpp",
+  "friendlyName": "Shell++",
+  "description": "Execute bash script",
+  "author": "ms-devlabs",
+  "helpMarkDown": "[More Information](https://github.com/microsoft/Extension-UtilitiesPack)",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+        "sh"
+    ],
+  "version": {
+    "Major": "0",
+    "Minor": "3",
+    "Patch": "1"
+  },
+  "minimumAgentVersion": "2.144.0",
```

Should be  higher agent version then
```suggestion
  "minimumAgentVersion": "2.206.0",
```